### PR TITLE
The answer of my test-devops

### PR DESCRIPTION
**Pull request by Daryljocky**
Hi, I spent a weekend on this test. I did not used github action before. And I never create a api with API Spec before. They cost me some time. 
Please have a code review about my answer.
Look forward to your reply.

### CODE REVIEW

**Comment by Daryljocky** [comment]
Sorry, I forgot leave my name
李志豪/daryl.li

**Comment by kaleocheng** [comment]
@Daryljocky Thanks for your time for that. I will start to review them today

File: test6/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Test6
```
Comment (kaleocheng) [inline test6/README.md:1]: seems you didn't commit the Github action files in here

Comment (Daryljocky) [reply test6/README.md:1]: Sorry! My answer of test6 was put in .github/workflow. It was included in .gitignore.

image

I need to close this PR and resend a new PR.

**Pull request closed without merging**